common/lib/api-client/models.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
       move_agreed_by: '',
       cancellation_reason: '',
       cancellation_reason_comment: '',
+      prison_transfer_reason: '',
```

Comment:
Didn't know where to put this comment but...it might be worth adding a little piece of text in the commit body as to why this is needed to give some more context to this fix. Maybe what was happening before and what will happen know this has been added.